sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -66,13 +66,13 @@ mkdir $SYCL_HOME/build
 cd $SYCL_HOME/build
 
 cmake -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="X86" \
--DLLVM_EXTERNAL_PROJECTS="llvm-spirv;sycl" \
--DLLVM_ENABLE_PROJECTS="clang;llvm-spirv;sycl" \
+-DLLVM_EXTERNAL_PROJECTS="llvm-spirv;sycl;opencl-aot" \
+-DLLVM_ENABLE_PROJECTS="clang;llvm-spirv;sycl;opencl-aot" \
 -DLLVM_EXTERNAL_SYCL_SOURCE_DIR=$SYCL_HOME/llvm/sycl \
 -DLLVM_EXTERNAL_LLVM_SPIRV_SOURCE_DIR=$SYCL_HOME/llvm/llvm-spirv \
 $SYCL_HOME/llvm/llvm
 
-make -j`nproc` sycl-toolchain
+make -j`nproc` sycl-toolchain opencl-aot
```

Comment:
Since this tool is optional, I suggest to remove this target here and add some brief description below that if you want to use ahead of time compilation for the "list of supported devices here", then you need to additionally build `opencl-aot` targets.

Reply:
@AlexeySachkov does it look ok?

To use ahead-of-time compilation for the Intel&reg; processors, additionally build opencl-aot target:
  
1. add ```opencl-aot``` to ```-DLLVM_EXTERNAL_PROJECTS``` and ```-DLLVM_ENABLE_PROJECTS``` variables above
2. add ```opencl-aot``` to ```make``` (for Linux) or ```ninja``` (for Windows) commands above

For more, see [opencl-aot documentation](../../opencl-aot/README.md).